packages/http/test/http-decorators.test.ts

```diff
@@ -1272,5 +1272,20 @@ describe("http: decorators", () => {
         Visibility.Update | Visibility.Create | Visibility.Patch,
       );
     });
+
+    it("ensures all properties visible ane effective optionality skipped when parameterVisibility has no arguments", async () => {
+      const { test } = (await runner.compile(`
+      @parameterVisibility
+      @test op test(@path id: string, @body example: {}): void;
+      `)) as { test: Operation };
+      deepStrictEqual(
```

Comment:
It feels like we should specifically replicate the removal of `@visibility("read")` `@path` parameters from the request body, as this is specific behavior we are trying to protect - once we have figured out hhow to get rid of LegacyVisibility, we should also remove that test.

Reply:
Unfortunately it's not actually specific to `@path` parameters from the request body. I take your point that I should test the `@parameterVisibility()` behavior more extensively, though, and I will add some more tests before merging.

Reply:
I beefed up the test and validated much more of what the metadatainfo returns.